ui/src/app/nav.tsx

```diff
@@ -9,10 +9,21 @@ import { usePathname } from "next/navigation";
 import { useAuth } from "./auth/authContext";
 import UploadCSRAsidePanel from "./certificate_requests/asideForm";
 import UploadUserAsidePanel from "./users/asideForm";
+import { getStatus } from "./queries"
 import { ChangePasswordModalData, ChangePasswordModal, ChangePasswordModalContext } from "./users/components";
 
 export function SideBar({ activePath, sidebarVisible, setSidebarVisible }: { activePath: string, sidebarVisible: boolean, setSidebarVisible: Dispatch<SetStateAction<boolean>> }) {
+    const [status, setStatus] = useState<{ version: string } | null>(null);
     const auth = useAuth()
+
+    useEffect(() => {
+        const fetchStatus = async () => {
+            const statusData = await getStatus();
+            setStatus(statusData);
+        };
+        fetchStatus();
+    }, []);
```

Comment:
I would usually avoid useEffect when making queries because there are a ton of pitfalls. Here for example we will be refetching the version every time this component redraws since there is no dependencies listed.

We should use react-query, which honestly is way simpler to use and understand:
```
const queryClient = useQueryClient()
const query = useQuery({ queryKey: [], queryFn: getStatus })
const version = query.data?.version
```